app/__tests__/home.test.ts

```diff
@@ -1,26 +1,7 @@
-import { UITestBuilder } from '@antiwork/shortest';
+import { test } from '@antiwork/shortest';
 
 interface loginButton {
   url: string;
 }
 
-define('Validate Dasboard is accessible by users', async () => {
-  afterAll(async () => {
```

Comment:
We shouldn't have random console logs that don't test anything.

Reply:
Removed the logs in the same PR #100